docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,769 @@
+of the Matter accessory example to learn how Bluetooth LE advertising is enabled
+for the given example.
+
+### Step 3: Determine network pairing credentials
+
+You must provide the controller with network credentials that will be further
+used in the device commissioning procedure to configure the device with a
+network interface, such as Thread or Wi-Fi.
+
+#### Thread network credentials
+
+Fetch and store the current Active Operational Dataset from the Thread Border
+Router. Depending on if the Thread Border Router is running on Docker or
+natively on Raspberry Pi, execute the following commands:
+
+-   For Docker:
+
+    ```
+    sudo docker exec -it otbr sh -c "sudo ot-ctl dataset active -x"
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+-   For native installation:
+
+    ```
+    sudo ot-ctl dataset active -x
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+#### Wi-Fi network credentials
+
+The following Wi-Fi network credentials must be determined to commission the
+Matter accessory device to the Wi-Fi network in next steps:
+
+-   Wi-Fi SSID
+-   Wi-Fi password
+
+Matter specification does not define how the Thread or Wi-Fi credentials are
+obtained by Controller.
+
+For example, for Thread, instead of fetching datasets directly from the Thread
+Border Router, you might also use a different out-of-band method.
+
+For Wi-Fi, the steps required to determine the SSID and password may vary
+depending on the setup. For instance, you might need to contact your local Wi-Fi
+network administrator.
+
+### Step 4: Determine Matter accessory device's _discriminator_ and _setup PIN code_
+
+The controller uses a 12-bit value called _discriminator_ to discern between
+multiple commissionable device advertisements, as well as a 27-bit _setup PIN
+code_ to authenticate the device. You can find these values in the logging
+terminal of the device (for instance, UART). For example:
+
+```
+I: 254 [DL]Device Configuration:
+I: 257 [DL] Serial Number: TEST_SN
+I: 260 [DL] Vendor Id: 65521 (0xFFF1)
+I: 263 [DL] Product Id: 32768 (0x8000)
+I: 267 [DL] Hardware Version: 1
+I: 270 [DL] Setup Pin Code: 20202021
+I: 273 [DL] Setup Discriminator: 3840 (0xF00)
+I: 278 [DL] Manufacturing Date: (not set)
+I: 281 [DL] Device Type: 65535 (0xFFFF)
+```
+
+In above printout, the _discriminator_ is 3840 (0xF00) and the _setup PIN code_
+is equal to 20202021 accordingly.
+
+### Step 5: Commission Matter accessory device into existing network
+
+#### Commissioning into Thread network over Bluetooth LE
+
+To commission the device to the existing Thread network run the following
```

Comment:
But do we have a Thread network at this point? So far, we've only prepared one device and determined its credentials?

Reply:
The assumption is that the Thread network is up and running, so that would probably classify as a general requirement or dedicated step, but I doubt that we should describe the OTBR setup as it relies on Open Thread specific solution. Following that we would also need to describe how to setup the WiFi network.

Reply:
If a Thread network is required, we need an entry under Requirements or a step that mentions setting the network up.

Reply:
Ok, I will add a separate step for that.